wdl2cwl/tests/wdl_files/tasks/tasks_reports.wdl

```diff
@@ -0,0 +1,538 @@
+version 1.0
+
+# Source: https://github.com/broadinstitute/viral-pipelines/blob/master/pipes/WDL/tasks/tasks_reports.wdl
```

Comment:
```suggestion
# Source: https://github.com/broadinstitute/viral-pipelines/blob/2a24ae82bf3c39dbbbfb8efdac0f7d276e780a4e/pipes/WDL/tasks/tasks_reports.wdl
```
To get a permalink, on almost any GitHub page hit the "y" key and then copy the updated location